ndk-glue/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+- Do not pass Looper `ident` as userdata pointer
```

Comment:
I think keeping the default formatting here is better
```
- **Breaking:** Looper `ident` not passed as userdata pointer anymore. If you are ...
```

Reply:
Yeah, better to not use the commit title here; but stil keeping the second paragraph. I prefer to use `in` instead of `as` to indicate the variable/member it's provided "into", though I better change userdata to `data` then to refer to the field.